assets/src/edit-story/components/canvas/layout.js

```diff
@@ -346,12 +340,8 @@ const PageArea = forwardRef(function PageArea(
     })
   );
   return (
-    <PageAreaFullbleedContainer
-      ref={fullbleedRef}
-      data-testid="fullbleed"
-      aria-label={__('Fullbleed area', 'web-stories')}
-      role="region"
-      overflow={overflow}
```

Comment:
I think a reference to this one is still needed in some places, but I'll fix that when merged. Good job here!